src/nni_manager/core/nniDataStore.ts

```diff
@@ -141,6 +141,7 @@ class NNIDataStore implements DataStore {
 
     public async getTrialJob(trialJobId: string): Promise<TrialJobInfo> {
         const trialJobs: TrialJobInfo[] = await this.queryTrialJobs(undefined, trialJobId);
+        assert(trialJobs.length <= 1);
```

Comment:
if trialJobs.length == 0, then asseration is right but will crash when take value "trialJobs[0]" ?

Reply:
Here is to avoid confict with this PR https://github.com/microsoft/nni/pull/1288/files. I don't exactly why this assert is added @chicm-ms